dtschema/schemas/chosen.yaml

```diff
@@ -167,6 +167,11 @@ properties:
       will assign devices in its usual manner, otherwise it will not try to
       assign devices and instead use them as they are configured already.
 
+  smbios3-entrypoint:
+    $ref: /schemas/types.yaml#/definitions/uint64
+    description:
+      The 64-bit physical address of the SMBIOSv3 entry point structure.
```

Comment:
I am not fluent in dt-schema, so I am not sure this matters, but I'll raise it just in case:  the address doesn't really need to be 64-bits. The SMBIOS3 entrypoint structure (which supersedes the SMBIOS entrypoint structure) was introduced to raise the max address of the SMBIOS table array beyond 4 GiB, but here we just need to capture the address of the entry point itself. IOW, maybe this should just be `types.yaml#/definitions/address` like the other fields?